pkg/process/config/config.go

```diff
@@ -281,15 +280,6 @@ func loadEnvVariables() {
 			config.Datadog.Set(variable.cfg, v)
 		}
 	}
-
-	if v := os.Getenv("DD_ORCHESTRATOR_ADDITIONAL_ENDPOINTS"); v != "" {
-		endpoints := make(map[string][]string)
-		if err := json.Unmarshal([]byte(v), &endpoints); err != nil {
-			log.Errorf(`Could not parse DD_ORCHESTRATOR_ADDITIONAL_ENDPOINTS: %v. It must be of the form '{"https://process.agent.datadoghq.com": ["apikey1", ...], ...}'.`, err)
-		} else {
-			config.Datadog.Set("orchestrator_explorer.orchestrator_additional_endpoints", endpoints)
-		}
-	}
```

Comment:
You may want to call this change out in the testing recommendation.

Reply:
There is an automated test for this env var already https://github.com/DataDog/datadog-agent/blob/2733839ba79090a9f0fa2d038b1ba56190527948/pkg/process/config/config_test.go#L376